### PR TITLE
Update package md to show c# sample with ogahandle

### PR DESCRIPTION
**Pull request by baijumeswani**

### CODE REVIEW

File: nuget/PACKAGE.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ You can call a high level `generate()` method to generate all of the output at o
 // See https://aka.ms/new-console-template for more information
 using Microsoft.ML.OnnxRuntimeGenAI;
 
-OgaHandle ogaHandle = new OgaHandle();
+using OgaHandle ogaHandle = new OgaHandle();
```
Comment (ajindal1) [inline nuget/PACKAGE.md:25]: Do we need to also provide some kind of warning to the user for the "using" keyword? @RyanUnderhill any thoughts?

Comment (baijumeswani) [reply nuget/PACKAGE.md:25]: What kind of warning did you have in mind? This is a sample that gets added inside the nuget published on nuget.org: https://www.nuget.org/packages/Microsoft.ML.OnnxRuntimeGenAI#readme-body-tab

Comment (ajindal1) [reply nuget/PACKAGE.md:25]: Discussed this with Ryan, it looks good.

**Review by ajindal1 — APPROVED** [review]

**Comment by natke** [comment]
Thank you for updating this @baiju!

**Pull request merged**